src/PharoThings-Devices-HD44780.package/PotLCD.class/instance/setPin.at..st

```diff
@@ -0,0 +1,4 @@
+initialization
+setPin: anObject at: aSymbol
+	gpioDic ifNil: [gpioDic := Dictionary new].
+	gpioDic add: aSymbol -> anObject 
```

Comment:
Why these pins are not managed as real instance variables?
I will comment in the used for concrete example

Reply:
Sure, them could be an instance variables. Is there some difference between both or is only to write less code (instead call the index (gpioDic at: #clockPin), call directly the instance variable clockPin?

Reply:
Generally speaking it is how we are supposed to model domains with objects. Instance variables declare explicit relationship between objects. Having a dictionary is like saying that your object can contain anything which I believe is not true for our case.
 
It is definitely less code. But the main goal is to understand the implementation. With instance variables we will immediately see in class definition how device is constructed, what pins it is using. Then all query tools will allow easily analyze code: simple variable references will show all places where pins are used.
Now it is required to know about gpioDict logic, to know the place where it is initialized and only having this information we can try to query code by senders of dictionary keys.